gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -389,6 +394,19 @@ private void addHDFSFiles(String jobFileList) throws IOException {
     }
   }
 
+  private void addHdfsJars(String hdfsJarFileList) throws IOException {
```

Comment:
This is more for Azkaban Gobblin job type as Gobblin has lots of dependency jars and each job execution will upload from local to hdfs which takes more than a minute. Having this already in HDFS dramatically reduces it.